man/git-sync.md

```diff
@@ -3,7 +3,7 @@ git-sync(1) -- Sync local branch with remote branch
 
 ## SYNOPSIS
 
-  `git sync` [--soft] [ &lt;remote&gt; &lt;branch&gt; ]
+  `git sync` [--soft] [--force] [ &lt;remote&gt; &lt;branch&gt; ]
```

Comment:
Let's generate other doc like https://github.com/tj/git-extras/tree/master/man#description

Reply:
> even better, include this as a pre-commit step

Any pull request to improve the doc generation is welcome!